### PR TITLE
adds missing INTIIAL_ADMIN_CODE

### PR DESCRIPTION
**Pull request by matti**

### CODE REVIEW

File: references/environment-variables.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - `MONGODB_URI`: MongoDB connection uri (required)
 - `VAULT_KEY`: secret key for the Kontena Vault (required)
 - `VAULT_IV`: initialization vector for the Kontena Vault (required)
+- `INTIIAL_ADMIN_CODE`: initial authorization code to obtain an access token for authenticating as the internal administrator (optional)
```
Comment (jnummelin) [inline references/environment-variables.md:8]: typo fix needed, it's `INITIAL_ADMIN_CODE` :)

**Review by jnummelin — CHANGES_REQUESTED** [review]

File: references/environment-variables.md (hunk 1 above)
Comment (kke) [reply references/environment-variables.md:8]: `rqeuire 'kotnena'`

**Comment by jnummelin** [comment]
@matti small typo fix needed

**Comment by matti** [comment]
clicked this #92 with github ui - took less than all the commenting here

**Pull request closed without merging**